app/views/people/tasks.html.erb

```diff
@@ -0,0 +1,34 @@
+<h1> List of tasks for <%= @person.name %> </h1>
+
+<table>
+  <thead>Task</thead>
+
+  <tr>
+    <th> ID </th>
```

Comment:
Watch for consistency with your white space on either side of these header list items